contracts/price-providers/V2TokenCollateralPeggedPriceProviderFallback.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity 0.5.8;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-eth/contracts/math/SafeMath.sol";
+import "openzeppelin-eth/contracts/math/Math.sol";
+
+import "./PriceProviderFallback.sol";
+import "../interface/IMoCV2.sol";
+import "../interface/IPriceProvider.sol";
+
+/**
+  @notice gets the technical price from V2 Protocol and covert it to pegged price
```

Comment:
I think a real example, for example using Rif, RifUSD, RifPRO, DOC in the description would make each PP more simple to understand and identify.